### PR TITLE
util: correct comments mistake

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: util/filesort/filesort.go [hunk 1]
```diff
@@ -92,12 +92,12 @@ func (rh *rowHeap) Less(i, j int) bool {
 	return ret
 }
 
-// Push implements heap.Interface Push interface.
+// Push push an element into rowHeap
```
Comment (hanfei1991) [inline util/filesort/filesort.go]: pushes

File: util/filesort/filesort.go [hunk 2]
```diff
@@ -92,12 +92,12 @@ func (rh *rowHeap) Less(i, j int) bool {
...
 func (rh *rowHeap) Push(x interface{}) {
 	rh.ims = append(rh.ims, x.(*item))
 }
 
-// Push implements heap.Interface Pop interface.
+// Pop pop the last element from rowHeap
```
Comment (hanfei1991) [inline util/filesort/filesort.go]: pops

File: util/filesort/filesort.go [hunk 3]
```diff
@@ -295,7 +295,7 @@ func (fs *FileSorter) closeAllFiles() error {
 	return nil
 }
 
-// Perform full in-memory sort.
+// internalSort perform full in-memory sort.
```
Comment (hanfei1991) [inline util/filesort/filesort.go]: performs

File: util/filesort/filesort.go [hunk 4]
```diff
@@ -314,7 +314,7 @@ func (fs *FileSorter) internalSort() (*comparableRow, error) {
 	return nil, nil
 }
 
-// Perform external file sort.
+// externalSort perform external file sort.
```
Comment (hanfei1991) [inline util/filesort/filesort.go]: ditto

File: util/filesort/filesort.go [hunk 5]
```diff
@@ -412,7 +412,7 @@ func (fs *FileSorter) openAllFiles() error {
 	return nil
 }
 
-// Fetch the next row given the source file index.
+// fetchNextRow fetchs the next row given the source file index.
```
Comment (hanfei1991) [inline util/filesort/filesort.go]: fetches

File: util/types/mytime.go [hunk 6]
```diff
@@ -209,7 +209,7 @@ func calcWeekday(daynr int, sundayFirstDayOfWeek bool) int {
 type weekBehaviour uint
 
 const (
-	// If set, Sunday is first day of week, otherwise Monday is first day of week.
+	// weekBehaviourMondayFirst set Monday is first day of week; otherwise Sunday is first day of week
```
Comment (hanfei1991) [inline util/types/mytime.go]: sets  Monday as

File: util/types/time.go [hunk 7]
```diff
@@ -2252,7 +2254,7 @@ func monthNumeric(t *mysqlTime, input string, ctx map[string]int) (string, bool)
 	return rem, false
 }
 
-// 0th 1st 2nd 3rd ...
+//  dayOfMonthWithSuffix return different suffix according t which day. i.e. 0 return th. 1 return st.
```
Comment (hanfei1991) [inline util/types/time.go]: returns

File: util/filesort/filesort.go (hunk 2 above)
Comment (XuHuaiyu) [inline util/filesort/filesort.go]: add `.` at the end of this comment.

File: util/encrypt/aes.go [hunk 8]
```diff
@@ -85,7 +85,7 @@ func newECBDecrypter(b cipher.Block) cipher.BlockMode {
 	return (*ecbDecrypter)(newECB(b))
 }
 
-// Padding using PKCS7
+// pkcs7Padding padds data using PKCS7
```
Comment (XuHuaiyu) [inline util/encrypt/aes.go]: add `.` at the end of this comment.

Comment (XuHuaiyu) [inline util/encrypt/aes.go]: s/padds/pads

File: util/filesort/filesort.go (hunk 1 above)
Comment (XuHuaiyu) [inline util/filesort/filesort.go]: add `.` at the end of this comment.

File: util/types/mydecimal.go [hunk 9]
```diff
@@ -419,15 +416,15 @@ func (d *MyDecimal) Shift(shift int) error {
 		return nil
 	}
 	var (
-		/* index of first non zero digit (all indexes from 0) */
+		// digitBegin is index of first non zero digit (all indexes from 0)
```
Comment (XuHuaiyu) [inline util/types/mydecimal.go]: add `.` at the end of this comment.

File: util/types/mydecimal.go [hunk 10]
```diff
@@ -419,15 +416,15 @@ func (d *MyDecimal) Shift(shift int) error {
...
 		digitBegin int
-		/* index of position after last decimal digit */
+		// digitEnd is index of position after last decimal digit
```
Comment (XuHuaiyu) [inline util/types/mydecimal.go]: add `.` at the end of this comment.

File: util/types/mydecimal.go [hunk 11]
```diff
@@ -419,15 +416,15 @@ func (d *MyDecimal) Shift(shift int) error {
...
 		digitEnd int
-		/* index of digit position just after point */
+		// point is index of digit position just after point
```
Comment (XuHuaiyu) [inline util/types/mydecimal.go]: add `.` at the end of this comment.

File: util/types/time.go [hunk 12]
```diff
@@ -661,6 +661,7 @@ func ParseYear(str string) (int16, error) {
 	return y, nil
 }
 
+// adjustYear will adjust year according to y
```
Comment (XuHuaiyu) [inline util/types/time.go]: s/will adjust/adjusts
add `.` at the end of this comment.

File: util/types/time.go [hunk 13]
```diff
@@ -986,6 +987,7 @@ func getTime(num int64, tp byte) (Time, error) {
 	return t, errors.Trace(err)
 }
 
+// parseDateTimeFromNum parses date time from num
```
Comment (XuHuaiyu) [inline util/types/time.go]: add `.` at the end of this comment.

File: util/encrypt/aes.go [hunk 14]
```diff
@@ -94,7 +94,7 @@ func pkcs7Padding(ciphertext []byte, blockSize int) []byte {
 	return append(ciphertext, padtext...)
 }
 
-// UnPadding using PKCS7
+// pkcs5UnPadding unPadd data using PKCS7
```
Comment (XuHuaiyu) [inline util/encrypt/aes.go]: s/unPadd/unpads

Comment (XuHuaiyu) [inline util/encrypt/aes.go]: add `.` at the end of this comment.

File: util/filesort/filesort.go [hunk 15]
```diff
@@ -42,7 +42,7 @@ type item struct {
 	value *comparableRow
 }
 
-// Min-heap of comparableRows
+// rowHeap maintains a min-heap porperty of comparableRows
```
Comment (XuHuaiyu) [inline util/filesort/filesort.go]: s/porperty/property
add `.` at the end of this comment.

File: util/types/time.go [hunk 16]
```diff
@@ -1354,7 +1356,7 @@ func extractSingleTimeValue(unit string, format string) (int64, int64, int64, go
 	return 0, 0, 0, 0, errors.Errorf("invalid singel timeunit - %s", unit)
 }
 
-// Format is `SS.FFFFFF`.
+// extractSecondMicrosecond extracts second microsecond from a string and its format is `SS.FFFFFF`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: 1. s/second microsecond/second and microsecond
2. s/and its format is/of which format is `SS.FFFFFF`

File: util/types/time.go [hunk 17]
```diff
@@ -1374,7 +1376,7 @@ func extractSecondMicrosecond(format string) (int64, int64, int64, gotime.Durati
 	return 0, 0, 0, gotime.Duration(seconds)*gotime.Second + gotime.Duration(microseconds)*gotime.Microsecond, nil
 }
 
-// Format is `MM:SS.FFFFFF`.
+// extractMinuteMicrosecond extracts minutes microsecond from a string and its format is `MM:SS.FFFFFF`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: ditto.

File: util/types/time.go [hunk 18]
```diff
@@ -1394,7 +1396,7 @@ func extractMinuteMicrosecond(format string) (int64, int64, int64, gotime.Durati
 	return 0, 0, 0, gotime.Duration(minutes)*gotime.Minute + value, nil
 }
 
-// Format is `MM:SS`.
+// extractMinuteSecond extracts minutes second from a string and its format is `MM:SS`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: ditto.

File: util/types/time.go [hunk 19]
```diff
@@ -1414,7 +1416,7 @@ func extractMinuteSecond(format string) (int64, int64, int64, gotime.Duration, e
 	return 0, 0, 0, gotime.Duration(minutes)*gotime.Minute + gotime.Duration(seconds)*gotime.Second, nil
 }
 
-// Format is `HH:MM:SS.FFFFFF`.
+// extractHourMicrosecond extracts hour microsecond from a string and its format is `HH:MM:SS.FFFFFF`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: ditto.

File: util/types/time.go [hunk 20]
```diff
@@ -1439,7 +1441,7 @@ func extractHourMicrosecond(format string) (int64, int64, int64, gotime.Duration
 	return 0, 0, 0, gotime.Duration(hours)*gotime.Hour + gotime.Duration(minutes)*gotime.Minute + value, nil
 }
 
-// Format is `HH:MM:SS`.
+// extractHourSecond extracts hour second from a string and its format is `HH:MM:SS`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: ditto.
the same as line 1469/1489/1509/1529/1549/1569

File: util/filesort/filesort.go (hunk 1 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

File: util/filesort/filesort.go (hunk 2 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

File: util/filesort/filesort.go (hunk 3 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

File: util/filesort/filesort.go (hunk 4 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

File: util/filesort/filesort.go (hunk 5 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

File: util/types/mytime.go (hunk 6 above)
Comment (zhexuany) [inline util/types/mytime.go]: DONE

File: util/types/time.go (hunk 7 above)
Comment (zhexuany) [inline util/types/time.go]: DONE

File: util/filesort/filesort.go (hunk 15 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE.

File: util/encrypt/aes.go (hunk 8 above)
Comment (zhexuany) [inline util/encrypt/aes.go]: DONE

File: util/types/time.go (hunk 12 above)
Comment (zhexuany) [inline util/types/time.go]: DONE

File: util/types/time.go (hunk 13 above)
Comment (zhexuany) [inline util/types/time.go]: DONE

**Comment by zhexuany** [comment]
@XuHuaiyu PTAL

File: util/filesort/filesort.go [hunk 21]
```diff
@@ -559,7 +559,7 @@ func (w *Worker) input(row *comparableRow) {
 	}
 }
 
-// Flush the buffer to file if it is full.
+// flushToFile flushs the buffer to file if it is full.
```
Comment (coocood) [inline util/filesort/filesort.go]: flushes

File: util/types/time.go [hunk 22]
```diff
@@ -661,6 +661,7 @@ func ParseYear(str string) (int16, error) {
 	return y, nil
 }
 
+// adjustYear will adjusts year according to y
```
Comment (coocood) [inline util/types/time.go]: remove `will`

Comment (zhexuany) [inline util/types/time.go]: DONE

File: util/filesort/filesort.go (hunk 21 above)
Comment (zhexuany) [inline util/filesort/filesort.go]: DONE

**Comment by coocood** [comment]
LGTM

File: util/types/datum_eval.go [hunk 23]
```diff
@@ -547,7 +547,7 @@ func ComputeRightShift(sc *variable.StatementContext, a, b Datum) (d Datum, err
 	return d, nil
 }
 
-// covertNonIntegerToUint64 coverts a non-integer to an uint64
+// convertNonInt2RoundUint64 coverts a non-integer to an uint64
```
Comment (XuHuaiyu) [inline util/types/datum_eval.go]: s/coverts/converts

File: util/types/time.go [hunk 24]
```diff
@@ -661,6 +661,7 @@ func ParseYear(str string) (int16, error) {
 	return y, nil
 }
 
+// adjustYear adjusts year according to y
```
Comment (XuHuaiyu) [inline util/types/time.go]: add `.` at the end.

File: util/types/time.go [hunk 25]
```diff
@@ -1354,7 +1356,7 @@ func extractSingleTimeValue(unit string, format string) (int64, int64, int64, go
 	return 0, 0, 0, 0, errors.Errorf("invalid singel timeunit - %s", unit)
 }
 
-// Format is `SS.FFFFFF`.
+// extractSecondMicrosecond extracts second and microsecond from a string and its format is `SS.FFFFFF`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: s/and its format is/of which format is

File: util/types/time.go [hunk 26]
```diff
@@ -1394,7 +1396,7 @@ func extractMinuteMicrosecond(format string) (int64, int64, int64, gotime.Durati
 	return 0, 0, 0, gotime.Duration(minutes)*gotime.Minute + value, nil
 }
 
-// Format is `MM:SS`.
+// extractMinuteSecond extracts minutes and second from a string and its format is `MM:SS`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: s/minutes/minute

File: util/types/time.go [hunk 27]
```diff
@@ -1484,7 +1486,7 @@ func extractHourMinute(format string) (int64, int64, int64, gotime.Duration, err
 	return 0, 0, 0, gotime.Duration(hours)*gotime.Hour + gotime.Duration(minutes)*gotime.Minute, nil
 }
 
-// Format is `DD HH:MM:SS.FFFFFF`.
+// extractDayMicrosecond extracts data and microsecond from a string and its format is `DD HH:MM:SS.FFFFFF`.
```
Comment (XuHuaiyu) [inline util/types/time.go]: s/data/day

File: util/types/time.go [hunk 28]
```diff
@@ -2252,7 +2254,7 @@ func monthNumeric(t *mysqlTime, input string, ctx map[string]int) (string, bool)
 	return rem, false
 }
 
-// 0th 1st 2nd 3rd ...
+//  dayOfMonthWithSuffix returns different suffix according t which day. i.e. 0 return th. 1 return st.
```
Comment (XuHuaiyu) [inline util/types/time.go]: what does according t which day mean...

File: util/types/time.go (hunk 25 above)
Comment (zhexuany) [inline util/types/time.go]: "its" represents a string. Not need replace by of which.

File: util/types/time.go (hunk 24 above)
Comment (zhexuany) [inline util/types/time.go]: DONE

File: util/types/time.go (hunk 27 above)
Comment (zhexuany) [inline util/types/time.go]: DONE

File: util/types/time.go (hunk 28 above)
Comment (zhexuany) [inline util/types/time.go]: should be "t being which day"

**Comment by shenli** [comment]
LGTM

**Pull request merged**